docs/webhooks/11-Safelist-IPs.md

```diff
@@ -0,0 +1,36 @@
+---
+tags: [webhooks]
+---
+> Notice: This safelist of IP addresses will take effect April 28th, 2022 - 14:00 UTC
+
+# Webhook Notifications
+
+If you are consuming Generic Webhooks from PagerDuty, use these IPs to make sure make sure they originate from a known PagerDuty Safelist IP.
+
+
+The full list of IP addresses that our webhook notifications may come from is:
```

Comment:
Does this list include the current IP addresses or only the static addresses that will be active after the cutover?

Reply:
If the latter, we likely want to include some language that describes the situation: 

* The `/webhook_ips` is the authoritative source until 'x' date.
* This list is the authoritative source after 'x' date.
* If you want total continuity in your firewall, you will need to safelist both sets of ip addresses some time before 'x' date.

Reply:
Complicated - this list will have _some_ of the IPS from `/webhook_ips` because the OWT NAT Gateway list IPs are in there. Maybe we should just put that language in anyways?